hofx/stage_bkg.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
```

Comment:
unless this is an executable (it should not), we should avoid adding the interpreter in .py files.  
That way one can do `python script.py` or `python3 script.py` or `/path/to/your/fav/python script.py`

Reply:
So I guess this is a high level philosophical question. Should a script like this be executable? Are you arguing that if it ends in .py we should treat it as 'python script.py' as opposed to treating it like a shell script? People rarely do 'bash script.sh'.

Reply:
Ok I see what you're saying and yes the intention long term (I think) is for most of this to be 'imported'. Will remove the shebang.